playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart

```diff
@@ -142,7 +144,18 @@ class ExamplesLoader {
   }
 
   Future<void> _loadOne(ExampleLoader loader) async {
-    final example = await loader.future;
+    Example example;
+    try {
+      example = await loader.future;
+    } on Exception catch (_) {
+      throw Exception(
```

Comment:
Would be nice to have `ExampleLoadingException` class with `token` as a property. It's not the loader's job to be translation-aware.